### PR TITLE
Added kahoot logo

### PR DESCRIPTION
**Pull request by nishd8**
**Issue: #3598**
**Alexa rank: 3,150**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by nishd8** [comment]
#3598

**Review by runxel — COMMENTED** [review]
Looks good, you just need to delete that one extra icon file!

File: kahoot.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...9.746 9.746 0 01-.553-.217 15.74 15.74 0 00-.595-.225 16.023 16.023 0 01-.659-.255c-.26-.12-.302-.115-.493.068l-.617.587-1.186 1.127c-.408.387-.808.77-.893.851-.085.08-.506.485-.936.893-.433.409-.986.936-1.233 1.175-.246.234-.897.855-1.446 1.374A56.95 56.95 0 004.67 10.14c-.072.08-.077-.128-.077-4.472V1.112l-.093.004c-.055 0-.23.043-.396.085zM17.216.066c0 .038.042.25.092.474.111.478.197.886.336 1.541.2.949.46 2.144.514 2.371.032.127.121.543.2.925.078.382.224 1.056.317 1.503.097.444.264 1.226.375 1.735.11.509.271 1.272.36 1.696.09.424.175.813.193.867.018.054.1.436.182.848.079.413.16.786.179.829.017.042.103.432.196.867.15.74.314 1.496.735 3.45.093.443.236 1.098.31 1.453.118.563.19.786.233.72.003-.007.07-.492.146-1.075.078-.582.153-1.102.168-1.156.017-.054.096-.59.174-1.195.083-.605.161-1.16.179-1.233.018-.073.107-.682.196-1.35.09-.666.172-1.24.182-1.271.01-.031.1-.62.2-1.31.096-.69.253-1.793.35-2.448l.353-2.448c.097-.69.186-1.307.2-1.368.064-.317.385-2.618.4-2.853l.014-.27-.635-.127c-.35-.073-1.046-.212-1.546-.316-.5-.104-1.224-.25-1.606-.328-.381-.077-.849-.173-1.035-.212a42.13 42.13 0 00-.945-.192c-.332-.066-.636-.135-.668-.154A.48.48 0 0017.358 0c-.11 0-.142.015-.142.066z"/></svg>
```
[the remark this answers is not in the document]
Comment (nishd8) [reply kahoot.svg:1]: Which folder is it?

[the remark this answers is not in the document]
Comment (nishd8) [reply kahoot.svg]: Done

**Comment by runxel** [comment]
Great! Thanks for you work! 💯

**Comment by PeterShaggyNoble** [comment]
The name of this one should be stylised as "Kahoot!". As the exclamation mark has no bearing on the filename (see "Yahoo!", for example), this _shouldn't_ constitute a breaking change but, as I've been away for a while, can one of the other @simple-icons/maintainers confirm before I open a PR?

**Comment by ericcornelissen** [comment]
> The name of this one should be stylised as "Kahoot!". As the exclamation mark has no bearing on the filename (see "Yahoo!", for example), this _shouldn't_ constitute a breaking change but, as I've been away for a while, can one of the other @simple-icons/maintainers confirm before I open a PR?

Good point @PeterShaggyNoble, this should be a backwards compatible change since 1) (as you point out) the filename (and by extension slug) doesn't change and 2) in [the `get` functionality](https://github.com/simple-icons/simple-icons/blob/c6bab5940d67a57d83572cf91c12957618830986/scripts/templates/index.js#L3-L17) the `targetName` "Kahoot" (for the current name) will still work because it will match the slug (after normalization) 👍 

Whenever this gets done, the next release should be merged carefully as [the release action](https://github.com/simple-icons/release-action) will probably see it as a non-backwards compatible change and version bump to v5!

**Pull request merged**